tests/conformance_ruby/tags/raw_tag_test.rs

```diff
@@ -40,7 +39,7 @@ fn test_open_tag_in_raw() {
     );
     assert_template_result!(
         " test {% raw %} {% endraw %}",
-        "{% raw %} test {% raw %} {% {% endraw %}endraw %}",
+        "{% raw %} test {% raw %} {% endraw %}{% endraw %}",
```

Comment:
Please revert, these tests should match Ruby to ensure we are compliant.  Besides, I don't see why the new version should pass.  It should end processing at the first `endraw`, causing the second `endraw` to fail.

And boy is that a clever way of printing out the raw tag, breaking it up into
- ` test {% raw %} {%`
- `endraw %}`

Reply:
Another thing I just noticed:
Are `}}` and `%}` tokens supposed to be accepted (assuming there is no matching `{%`)? As in, is `This input which is just raw text with %} in the middle` accepted in ruby liquid?

Because that can be easily fixed. I just assumed that a `}}` or `%}` would be errors when alone.

Reply:
From my understanding, all processing should be disabled within a `raw` until `endraw` is encountered, including stray tokens, new, unclosed `raw`, etc.

Reply:
So this is where all my confusion came from. My eyes were parsing the input string incorrectly.

I guess `%}` and `}}` should also be allowed, having in mind both this test and #320